test/components/navigationBarTest.js

```diff
@@ -322,10 +325,135 @@ describe('navigationBar', function () {
           )
         })
         .windowByUrl(Brave.browserWindowUrl)
+        .waitForExist(urlbarIcon)
         .click(urlbarIcon)
-        .waitForExist(urlbarIcon + '.fa-unlock')
         .waitForVisible('[data-l10n-id="mixedConnection"]')
-        .keys('\uE00C')
+        .waitForVisible('.denyRunInsecureContentWarning')
+        .waitForVisible(dismissDenyRunInsecureContentButton)
+        .waitForVisible(denyRunInsecureContentButton)
+        .click(denyRunInsecureContentButton)
+        .tabByUrl(this.page1Url)
+        .waitUntil(() => {
+          return this.app.client.getCssProperty('body', 'background-color').then((color) =>
+            color.value === 'rgba(128,128,128,1)'
+          )
+        })
+        .windowByUrl(Brave.browserWindowUrl)
+        .click(urlbarIcon)
+        .waitForExist(urlbarIcon + '.fa-lock')
+    })
+    it('Limit effect of running insecure content in frame', function * () {
```

Comment:
this test is timing out: https://travis-ci.org/brave/browser-laptop/builds/158405114#L3620